web/src/search/searchFilterSuggestions.ts

```diff
@@ -102,8 +102,7 @@ export const searchFilterSuggestions: SearchFilterSuggestions = {
         ),
     },
     type: {
-        default: 'code',
-        values: [{ value: 'code' }, { value: 'diff' }, { value: 'commit' }, { value: 'symbol' }].map(
+        values: [{ value: 'diff' }, { value: 'commit' }, { value: 'symbol' }, { value: 'file' }].map(
```

Comment:
This should include `path` too

```suggestion
        values: [{ value: 'diff' }, { value: 'commit' }, { value: 'symbol' }, { value: 'file' }, { value: 'path' }].map(
```